Upstatement/skela-wp-theme
# Update php code style

## Changes

This PR proposes to update our PHP code style to align with the [WordPress standards](https://github.com/WordPress/WordPress-Coding-Standards). It also aligns all of our PHP files to this new standard.

I'm sorry in advance for all of the strict linting rules.

## How To Test

1. Pull down this branch.
1. Run `composer update` to install the WordPress coding standards sniffs.
1. Make sure there are no linting errors.

## PHP linting and VSCode

I use the [phpcs extension](https://marketplace.visualstudio.com/items?itemName=ikappas.phpcs) for VSCode to help display linting errors. The instructions in that package are pretty good to get up and running.
